lib/CModHandler.cpp

```diff
@@ -465,7 +458,7 @@ void CContentHandler::init()
 	handlers.insert(std::make_pair("heroes", ContentTypeHandler(VLC->heroh, "hero")));
 	handlers.insert(std::make_pair("spells", ContentTypeHandler(VLC->spellh, "spell")));
 	handlers.insert(std::make_pair("skills", ContentTypeHandler(VLC->skillh, "skill")));
-	handlers.insert(std::make_pair("templates", ContentTypeHandler((IHandlerBase *)VLC->tplh, "template")));
+	handlers.insert(std::make_pair("templates", ContentTypeHandler(reinterpret_cast<IHandlerBase *>(VLC->tplh), "template")));

```

Comment:
Remove cast and instead add 
`#include "mapObjects/CObjectClassesHandler.h"`

Reply:
cast removed, include already there